bin/install-wordpress.sh

```diff
@@ -90,6 +90,24 @@ if [ "$CURRENT_URL" != "http://localhost:$HOST_PORT" ]; then
 	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI option update siteurl "http://localhost:$HOST_PORT" --quiet
 fi
 
+# Configure site constants.
+echo -e $(status_message "Configuring site constants...")
```

Comment:
There is a lot of code duplication here. Commits which simplify it are warmly welcomed :)

Reply:
We could use macro-like behaviour with the following:

```bash
$ A=hello; B=salut
$ VAR=A; eval echo \$$VAR
hello
$ VAR=B; eval echo \$$VAR
salut
```